test/EFCore.PG.FunctionalTests/Query/NodaTimeQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,2028 @@
+﻿using NodaTime;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query;
+
+public class NodaTimeQueryNpgsqlTest : QueryTestBase<NodaTimeQueryNpgsqlTest.NodaTimeQueryNpgsqlFixture>
```

Comment:
Nope, I moved them as-is - they were previously in a separate project (EFCore.PG.NodaTime.FunctionalTests), and I just integrated them into the regular tests project. It was this way because previously we set up NodaTime on the global type mapper, which interfered with the non-NodaTime tests, and each project is run in its own process.

But now everything is scoped to the data source, so no more isolation problems :)